biome.jsonc

```diff
@@ -6,7 +6,7 @@
     "useIgnoreFile": true,
     "defaultBranch": "main"
   },
-  "files": { "ignoreUnknown": true, "ignore": [] },
+  "files": { "ignoreUnknown": true, "ignore": ["*.test.tsx"] },
```

Comment:
Let's not do this if there are too many cases - better to ignore one lint rule than disable all for some files.